pkg/operation/botanist/component/logging/kuberbacproxy/kube_rbac_proxy.go

```diff
@@ -50,8 +50,8 @@ type Values struct {
 	Namespace string
 }
```

Comment:
`Client` and `Namespace` should not be part of Values. Values should include properties you would put into a helm values.yaml.
[CoreDNS provide a good example](https://github.com/gardener/gardener/blob/bfaf5403fefbe7ca379cc0e2402b535c2052df71/pkg/operation/botanist/component/coredns/coredns.go#L85-L126) how Values should look like.
For `kubeRBACProxy` we would not need a Values structure at all. `client` and `namespace` should be part of type `kubeRBACProxy`.